test/tests/functional/pbs_alps_release_tunables.py

```diff
@@ -44,7 +44,7 @@
 
 def get_epoch(msg):
     # Since its a log message split on ';' to get timestamp
-    a = time.strptime(msg.split(';')[0], "%m/%d/%Y %H:%M:%S")
+    a = time.strptime(msg.split(';')[0], "%m/%d/%Y %H:%M:%S.%f")
```

Comment:
Use LogUtil's convert_date_time().  This handles microseconds properly.  time's %f is broken.  You need to use datetime which convert_date_time() does.